packages/services-store/src/actions/profileAPI.ts

```diff
@@ -34,19 +46,24 @@ export function setUserStatus(status: UserStatus) {
   };
 }
 
-export type ProfileAPIActions = ReturnType<typeof setMonitorUserStatusError> | ReturnType<typeof setUserStatus>;
+export type ProfileAPIActions =
+  | ReturnType<typeof setErrorRequestUserStatus>
+  | ReturnType<typeof setRequestUserStatusInProgress>
+  | ReturnType<typeof setUserStatus>;
 
 /**
- * @returns A Redux action that monitors user authentication status.
+ * @returns A Redux action that gets user authentication status.
  */
-export function monitorUserStatus(): ThunkAction<void, { profileAPI: ProfileAPIState }, void, ProfileAPIActions> {
-  return dispatch => {
-    ProfileAPI.monitorUserStatus((error: Error, status: UserStatus) => {
-      if (error) {
-        dispatch(setMonitorUserStatusError(error));
-      } else {
-        dispatch(setUserStatus(status));
-      }
-    });
+export function getUserStatus(): ThunkAction<Promise<UserStatus>, { profileAPI: ProfileAPIState }, void, ProfileAPIActions> {
```

Comment:
I should have told you earlier (my apologies), Redux actions in this category takes `load*` name and we should align the naming strategy. Here's a reference: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.14.0/packages/services-store/src/actions/localeAPI.ts#L145-L164